[PR] Ajax comment form submission

philcable opened the pull request:
Fixes #20 @jeremyfelt #reviewmerge?

Commit:
Additional comment form parameters to remove unused output

Commit:
Allow comment template shortcode to accept content

Commit:
Add javascript to allow ajax comment submissions

Commit:
Add styles for added comment form markup

Commit:
Add docblock for comment form post-submission message

jeremyfelt commented:
I just rebased this against master and force pushed 😱  to resolve the merge conflict. :)

Reviewing now

jeremyfelt committed:
Skip moderator notification

There's some quirky behavior with emails, at least in a local
environment, so let's skip notification for now and just watch
the site for incoming emails.

We can work on a proper notification thing in the future.

jeremyfelt committed:
Remove the comments list from the main page

We'll show these elsewhere at a later time.

jeremyfelt commented:
@philcable Tossing this back to you for more review. 
- Disabled moderator notification to avoid possible email issues.
- Removed the list of comments on the main page.

philcable commented:
Looks good to me!
